Framework/jmaqs-appium/src/test/java/com/magenic/jmaqs/appium/AppiumDriverFactoryTest.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2019 (C) Magenic, All rights Reserved
+ */
+
+package com.magenic.jmaqs.appium;
+
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.remote.MobileCapabilityType;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ * The type Appium driver factory test.
+ */
+public class AppiumDriverFactoryTest {
+
+  private static final String MESSAGE = "Test method not implemented yet.";
```

Comment:
This would probably be better somewhere else like static classes for final strings.